ergo/platforms/metaculus/metaculus.py

```diff
@@ -67,12 +67,29 @@ class Metaculus:
         "interested": "upvoted_by",
     }
 
-    def __init__(self, username: str, password: str, api_domain: str = "www"):
+    def __init__(
+        self,
+        username: str = None,
+        password: str = None,
+        user_api_key: str = None,
+        org_api_key: str = None,
+        api_domain: str = "www",
+    ):
         self.user_id = None
         self.api_domain = api_domain
         self.api_url = f"https://{api_domain}.metaculus.com/api2"
         self.s = requests.Session()
-        self.login(username, password)
+        self.auth_method = "username_and_password"
+        self.user_api_key = user_api_key
+        self.org_api_key = org_api_key
+        if username and password:
+            self.login(username, password)
+        elif user_api_key and org_api_key:
+            self.auth_method = "api_keys"
+        else:
+            raise ValueError(
+                "Authentication method not provided. Please provide either username and password, or user and org API keys."
+            )
 
     def login(self, username, password):
```

Comment:
What should happen when a user calls `.login()` on a Metaculus object that was set up with API authentication?

Reply:
Fair point. I think it would be reasonable to switch to user/pass authentication, I will implement this.